packages/ui/src/components/molecules/Rating/styles.scss

```
@@ -29,10 +31,19 @@
   }
 
   [data-fs-rating-button] {
+    &[disabled] {
+      [data-fs-button-wrapper] {
+        background: transparent !important;
+        opacity: 0.5;
+      }
+    }
+
     color: unset;
```

```suggestion
     color: unset;

    &[disabled] [data-fs-button-wrapper] {
      background-color: transparent;
      &:hover {
        background-color: transparent;
      }
    }
```

Let's avoid using `!important` here. I think this will do it, removing background color on disabled. 
I didn't see any opacity requirement for disabled states in Figma. WDYT?

---

Done ✅ To be honest, I didn't even knew that was a figma with the disable styles. Do you have a link? I have access only to [this](https://www.figma.com/design/YXU6IX2htN2yg7udpCumZv/Reviews-%26-Ratings?node-id=1-958&p=f&t=hvdVQfzTmgTLIa32-0) review and ratings figma

---

I was looking in the same figma file as you, but I couldn’t find it either.

@renatamottam just double checking the disabled style should be have no background in both default and hover, correct?

---

that's it, fan!

---

Thank you!